.circleci/config.yml

```diff
@@ -14,6 +14,8 @@ jobs:
 
       - restore_cache:
           key: v1-macos-gems-{{ checksum "Gemfile" }}
+      - restore_cache:
```

Comment:
Move that down to before the test run maybe? No need to do it this early.

Reply:
Possible, although I put it next to the other restore_cache for consistency

Reply:
Thought behind that placement was that it "wraps" the step that is creating/using that cache.

Reply:
Hmm, good point. I changed it!